src/experiments.rs

```diff
@@ -42,6 +43,30 @@ string_enum!(pub enum CapLints {
     Forbid => "forbid",
 });
 
+/// A comma-separated list of capabilites possessed by an agent.
+#[derive(Default, Serialize, Deserialize)]
+#[serde(rename_all = "kebab-case")]
+pub struct Capabilities {
+    capabilities: Option<String>,
```

Comment:
Can we store capabilities internally as a list and then serialize them to a string when needed?